docs/apigateway.md

```diff
@@ -1,53 +1,105 @@
-# API Gateway (Experimental)
+# API Gateway
 
-[Web Actions](webactions.md) are out from experimental phase.
+OpenWhisk actions can benefit from being managed by API management.
 
-Web Actions allows you to invoke an action with HTTP methods other than POST and without the action's authorization API key.
+The API Gateway acts as a proxy to [Web Actions](webactions.md) and provides them with additional features features including HTTP method routing, security and rate limiting policies and more.
```

Comment:
I'm waiting on some security docs to be publish on apigateway so I can add more details, https://github.com/openwhisk/openwhisk-apigateway/pull/190
this will show to the user more info about api keys "security" in swagger spec.